### PR TITLE
JohnPapa Style Guide : Entity files

### PR DESCRIPTION
**Pull request by wmarques**
As suggested by @deepu105 in #2271, I've done entity files refactoring following [JohnPapa's rules](https://github.com/johnpapa/angular-styleguide/tree/master/a1).

I tested a sample app with 2 entities, in dev and prod profile and it was ok, with and without translations.

I followed most of the rules, I chose to ommit the [Y033](https://github.com/johnpapa/angular-styleguide/tree/master/a1#style-y033) (bindable members top) because most of controllers' functions are quite short, so for me it wasn't necessary to bind them.

### CODE REVIEW

File: generators/entity/templates/src/main/webapp/app/entities/_entity-management.state.js [hunk 1]
```diff
@@ -1,162 +1,170 @@
...
+                            templateUrl: 'app/entities/<%= entityFolderName %>/<%= entityFileName %>.html',
+                            controller: '<%= entityClass %>ManagementController',
+                            controllerAs: 'vm'
+                        }
                     },
-                    sort: {
-                        value: 'id,asc',
-                        squash: true
+                    <%_ if (pagination == 'pagination' || pagination == 'pager'){ _%>
+                    params: {
```
Comment (deepu105) [inline generators/entity/templates/src/main/webapp/app/entities/_entity-management.state.js]: is this correct? seems to be from another active PR?

File: generators/entity/templates/src/main/webapp/app/entities/_entity-management.state.js [hunk 2]
```diff
@@ -1,162 +1,170 @@
...
-                        controller: '<%= entityClass %>ManagementController'
-                    }
-                },
-                <%_ if (pagination == 'pagination' || pagination == 'pager'){ _%>
-                params: {
-                    page: {
-                        value: '1',
-                        squash: true
+    angular.module('<%=angularAppName%>')
+        .config(function ($stateProvider) {
```
Comment (deepu105) [inline generators/entity/templates/src/main/webapp/app/entities/_entity-management.state.js]: dont you have to inject the declaration as well?

Comment (deepu105) [inline generators/entity/templates/src/main/webapp/app/entities/_entity-management.state.js]: oops sorry ignore this, its a config block

**Comment by deepu105** [comment]
@wmarques this seems to have the commits from PR #2959 as well can you remove those commits plz

File: generators/entity/templates/src/main/webapp/app/entities/_entity-management.state.js (hunk 1 above)
Comment (wmarques) [inline generators/entity/templates/src/main/webapp/app/entities/_entity-management.state.js]: In this file I have just added the `controllerAs : 'vm'` lines.

Comment (deepu105) [inline generators/entity/templates/src/main/webapp/app/entities/_entity-management.state.js]: the diff shows more changes, but its correct compared to latest master so this is fine

**Comment by wmarques** [comment]
Yes sorry, it's because I created this branch from the one I've done in PR #2959 and not from master, my bad. 
I removed the first commit, now all the commits should concern entity files.

**Comment by deepu105** [comment]
much better ill test and merge

**Comment by deepu105** [comment]
can you correct the formatting of your IIFE PR as well, it has conflict as well so plz rebase

**Comment by wmarques** [comment]
It's done. Thanks for your help !

File: generators/entity/templates/src/main/webapp/app/entities/no-pagination-template.ejs [hunk 3]
```diff
@@ -1,23 +1,23 @@
...
-        $scope.openFile = DataUtils.openFile;
-        $scope.byteSize = DataUtils.byteSize;
-        <%_ } _%>
-        $scope.loadAll = function() {
-            <%= entityClass %>.query(function(result) {
-               $scope.<%= entityInstancePlural %> = result;
-            });
-        };<%_
-        if (searchEngine == 'elasticsearch') { _%>
+vm.<%= entityInstancePlural %> = [];
```
Comment (deepu105) [inline generators/entity/templates/src/main/webapp/app/entities/no-pagination-template.ejs]: incorrect formatting it should indent spaces more than current indent

File: generators/entity/templates/src/main/webapp/app/entities/no-pagination-template.ejs [hunk 4]
```diff
@@ -1,23 +1,23 @@
...
+<%_ if (fieldsContainBlob) { _%>
+    vm.openFile = DataUtils.openFile;
```
Comment (deepu105) [inline generators/entity/templates/src/main/webapp/app/entities/no-pagination-template.ejs]: these also shoul be indent more space than original

Comment (deepu105) [inline generators/entity/templates/src/main/webapp/app/entities/no-pagination-template.ejs]: i have corrected this

**Pull request merged**